Makefile

```diff
@@ -29,7 +29,7 @@ format: .venv
 .PHONY: format
 
 test: .venv
-	@$(VENV_PATH)/bin/tox -e py2.7,py3.7
+	@$(VENV_PATH)/bin/tox -e py2.7,py3.7,py3.6
```

Comment:
Please undo this, this target is only meant to test a few versions and to be fast, not all of them.

Reply:
Hm, I've thought that you want to test it for both versions of python. `aiocontextvars` not used in py3.7.

Reply:
@aCLr right, but `make test` doesn't run the whole testsuite, `make test-all` does. The test matrix you want is in `tox.ini`.

Reply:
Oh, I see, ok